src/basic/cgroup-util.c

```diff
@@ -2907,6 +2907,7 @@ bool fd_is_cgroup_fs(int fd) {
 static const char *const cgroup_controller_table[_CGROUP_CONTROLLER_MAX] = {
         [CGROUP_CONTROLLER_CPU] = "cpu",
         [CGROUP_CONTROLLER_CPUACCT] = "cpuacct",
+        [CGROUP_CONTROLLER_CPUSET] = "cpuset",
```

Comment:
(Github won't allow me comment a line out of the hunk's context, so to the line 2440 -- the comment is obsolete, let's remove references to particular controllers completely.)

Reply:
Good point, I've missed that comment. I'll fix it in separate commit.